stdlib/public/core/HashedCollections.swift.gyb

```diff
@@ -2523,7 +2523,7 @@ final internal class _Native${Self}StorageImpl<${TypeParameters}> {
 
   @_versioned
   internal var _capacity: Int {
-    return _body.capacity
+    return _assumeNonNegative(_body.capacity)
```

Comment:
I actually can't remember exactly, but there were some "< 0"-checks in llvm code.